bcgov/onroutebc
# Orv2 975

# Description

This PR is for frontend downloading permit PDF. 

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

1. Create a new application from the frontend.
2. Continue to complete the new application until see the success page.
3. Go to backend swagger UI to update the application's status to "ISSUED".
4. Go back to the success page and click "View Permits" button.
5. The permit PDF file is expected to be downloaded to the local machine for review.
6. Open the downloaded PDF, expecting no errors when opening the document.



## Checklist






## Further comments




---

Thanks for the PR!

Any successful deployments (not always required) will be available below.
[Backend - Vehicles](https://onroutebc-506-backend-vehicles.apps.silver.devops.gov.bc.ca/) available
[Backend - DMS](https://onroutebc-506-backend-dms.apps.silver.devops.gov.bc.ca/) available
[Frontend](https://onroutebc-506-frontend.apps.silver.devops.gov.bc.ca/) available

Once merged, code will be promoted and handed off to following workflow run.
[Main Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge-main.yml)
